### PR TITLE
Hexen autorun/novert

### LINKED ISSUE #885: Toggles for auto-run and vertical mouse movement in Hexen
### Background

Version of Crispy Doom: built locally from commit [78d0ebd](https://github.com/fabiangreffrath/crispy-doom/commit/78d0ebd518effedb753b736f434b32286a18f8b6)

Operating System and version: N/a

Game: Hexen

Any loaded WADs and mods (please include full command line): N/a

### Bug description

Observed behavior: at the moment Hexen is lacking these, while both Doom and Heretic have them. The only way to enable auto-run and disable vertical mouse movement is through the set-up tool. Not sure if there are other functionality keys missing from Hexen (or Heretic) but I guess they can also be mentioned here. I guess the set-up tool for Hexen could be restructured a little bit (are the weapon 5-8 keys really necessary considering vanilla Hexen only has 4 weapons for class?).

Expected behavior: not much to add, I'm mostly testing Crispy Heretic and Hexen and if I notice anything that could make for a nice improvement to bring them up to par with Crispy Doom I'll mention them.

### PR DESCRIPTION
**Pull request by rzhxiii**
This seemed to be pretty straightforward to implement, but since I'm a first timer, I have no idea if I'm doing any of this right so I gladly expect feedback, regardless of whether it's positive or negative.
Closes #885 .
All actual credit goes to [kraflab](https://github.com/kraflab), since I only ported their implementation for Heretic with some minor changes.

### CODE REVIEW

**Comment by mikeday0** [comment]
Everything looks good on my end! Thanks for taking care of this.

File: src/setup/keyboard.c [hunk 1]
```diff
@@ -230,7 +230,7 @@ static void ConfigExtraKeys(TXT_UNCAST_ARG(widget), TXT_UNCAST_ARG(unused))
         AddKeyControl(table, "Toggle vert. mouse", &key_togglenovert);
         AddKeyControl(table, "Quick Reverse", &key_reverse);
         }
-        else if (gamemission == heretic)
+        else if (gamemission == heretic || gamemission == hexen)
```
Comment (fabiangreffrath) [inline src/setup/keyboard.c:233]: It seems that Hexen is still missing the `key_alt_*` bindings which are also set in this section. Would you mind adding them as well in this PR?

Comment (rzhxiii) [reply src/setup/keyboard.c:233]: You're right, the binds show in the setup tool, but they don't work in game.
I'll take a look at it.

**Comment by fabiangreffrath** [comment]
Thank you very much!

**Pull request merged**